libraries/standard/http/include/http_client.h

```diff
@@ -112,7 +156,7 @@ typedef int32_t (* HTTPTransportSend_t )( HTTPNetworkContext_t * pContext,
  * @return The number of bytes read or a negative error code.
  */
 typedef int32_t (* HTTPTransportRecv_t )( HTTPNetworkContext_t * pContext,
-                                          const void * pBuffer,
+                                          void * pBuffer,
```

Comment:
Why a `void *` type instead of `uint8_t *` type for the buffer to store data in?

Reply:
Trying to match the MQTT specification.